internal/core/thirdparty/tantivy/tantivy-binding/src/util.rs

```diff
@@ -45,4 +46,23 @@ pub(crate) fn get_string_list(value: &json::Value, label: &str) -> Result<Vec<St
         }
     };
     Ok(str_list)
-}
\ No newline at end of file
+}
+
+pub(crate) fn get_stop_words_list(str_list:Vec<String>) -> Vec<String>{
```

Comment:
use &[String] as parameter may be better?

Reply:
Direct function param passing in rust mean pass owner of struct， not memory copy，in this function means the passing struct will free when get_stop_words_list finish.